AitGroup25Assignment/src/ai2018/group25/Group25_AS.java

```diff
@@ -70,7 +102,16 @@ public String getName() {
 	
 	@Override
 	public String printParameters() {
-		return "[paramter 1: " + parameter1 + " ]";
+		return "[paramter 1: " + reservation_value + " ]";
 	}
-
+	
+	private double what_reject(double normalized_time) {
+		if ( normalized_time <= starts_falling ) { 
+			return 1.0;
+		} else {
+			return ( -1/(1-starts_falling) )*normalized_time + ( 1/(1-starts_falling) ); // Still linear, transform into exponential

```

Comment:
It is more usefull to write javadoc above the function what it should do. Then this comment wouldn't be needed.